src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -148,14 +148,14 @@ public async Task GetAsync_SetCookieContainerMultipleCookies_CookiesSent()
 
             foreach (Cookie c in cookies)
             {
-                cookieContainer.Add(Configuration.Http.Http2RemoteEchoServer, c);
+                cookieContainer.Add(Configuration.Http.Http2SecureRemoteEchoServer, c);
```

Comment:
Why the change from `Http2RemoteEchoServer` to `Http2SecureRemoteEchoServer`?

Reply:
For this and other related changes

Reply:
Because it's actually used as HTTPS, currently wrongly named. In our case it also matters to use HTTPS protocol to reach correct port number, which is set by the environment variable.